uiowa/uiowa
# Feature: Design adjustments to admissions colleges page

Resolves #3255. 

# How to test

- `blt ds --site=admissions.uiowa.edu`
- `yarn workspace admissions_core build`
- `drush @admissions.local cr`
- Go to http://admissions.local.drupal.uiowa.edu/admin/structure/taxonomy/manage/colleges/add and add a College. 
- Reference that college http://admissions.local.drupal.uiowa.edu/node/981/edit?destination=/admin/content
- Verify that the college page looks similar to screenshot below:


<img width="1407" alt="Screen Shot 2021-03-22 at 9 53 38 AM" src="https://user-images.githubusercontent.com/1036433/112010151-0d518900-8af5-11eb-824e-8d354e0fba56.png">
